docs/tutorials/dataset_creation/point_maze_dataset.py

```diff
@@ -396,8 +395,7 @@ def __call__(
 
     obs, rew, terminated, truncated, info = collector_env.step(action)
 
-dataset = minari.create_dataset_from_collector_env(
-    collector_env=collector_env,
+dataset = env.create_dataset(
```

Comment:
to fix tests, `env` -> `collector_env` here

Reply:
done